satellite/accounting/tally/tally.go

```diff
@@ -31,21 +32,19 @@ type Service struct {
 	logger                  *zap.Logger
 	metainfo                *metainfo.Service
 	overlay                 *overlay.Service
-	limit                   int
-	ticker                  *time.Ticker
+	Loop                    sync2.Cycle
 	storagenodeAccountingDB accounting.StoragenodeAccounting
 	projectAccountingDB     accounting.ProjectAccounting
 	liveAccounting          live.Service
 }
 
 // New creates a new tally Service
-func New(logger *zap.Logger, sdb accounting.StoragenodeAccounting, pdb accounting.ProjectAccounting, liveAccounting live.Service, metainfo *metainfo.Service, overlay *overlay.Service, limit int, interval time.Duration) *Service {
+func New(logger *zap.Logger, sdb accounting.StoragenodeAccounting, pdb accounting.ProjectAccounting, liveAccounting live.Service, metainfo *metainfo.Service, overlay *overlay.Service, interval time.Duration) *Service {
```

Comment:
maybe outside the scope of this PR but wouldn't it be more consistent with other packages if this function were called `NewService` instead of just `New`?

Reply:
Yes, it also probably should be renamed to chore as well.

Reply:
Yes, it also probably should be renamed to chore as well.

Reply:
Also, yes, a separate PR makes more sense.